### PR TITLE
added a mode where mission control can control the hand 

### PR DESCRIPTION
**Pull request by Mchan2003**
issue #76

Added a hand mode called ManualHand where mission control can control the hand.

### CODE REVIEW

File: subsystem-arm/implementations/hand/hand-modes.hpp [hunk 1]
```diff
@@ -39,6 +40,17 @@ namespace sjsu::arm
...
         }
 
+        static hand_arguments ManualHand(hand_arguments arguments_)
+        {
+            hand_arguments temp_;
+            temp_.pinky_angle = arguments_.pinky_angle;
+            temp_.ring_ange = arguments_.ring_angle;
+            temp_.middle_angle = arguments_.middle_angle;
+            temp_.index_angle; = arguments_.index_angle;
+            temp_.thumb_angle = arguments_.thumb_angle;
```
Comment (Coreyboy1820) [inline subsystem-arm/implementations/hand/hand-modes.hpp:50]: Good job but for this more all you would have to do is return the thing you passed in, as long as all the conversions are handled somewhere else we should be okay.

File: subsystem-arm/test/hand-mode_test.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        SECTION("Should set angles to correct given manual mode angles")
+        {
+            hand_arguments original_commands {100, 100, 100, 100, 100, 100, 100, 100};
+            hand_arguments args = HandModes::ManualMode(original_commands);
+            CHECK(args.hand_args.pinky_angle == original_commands.hand_args.pinky_angle);
+            CHECK(args.hand_args.ring_angle == original_commands.hand_args.ring_angle);            
+            CHECK(args.hand_args.middle_angle == original_commands.hand_args.middle_angle);
+            CHECK(args.hand_args.index_angle == original_commands.hand_args.index_angle);
+            CHECK(args.hand_args.thumb_angle == original_commands.hand_args.thumb_angle);
```
Comment (Coreyboy1820) [inline subsystem-arm/test/hand-mode_test.cpp:18]: NICE!!! I love to see everyone making tests for there functions, good job!!

**Review by Coreyboy1820 — CHANGES_REQUESTED** [review]

File: subsystem-arm/implementations/hand/hand-modes.hpp [hunk 3]
```diff
@@ -1,47 +1,53 @@
...
+        {
+
+        }
+
+        static hand_arguments RestMode(hand_arguments arguments)
+        {
+
+        }
+
+
```
Comment (Coreyboy1820) [inline subsystem-arm/implementations/hand/hand-modes.hpp:47]: Don't need

File: subsystem-arm/implementations/hand/hand-modes.hpp [hunk 4]
```diff
@@ -1,47 +1,53 @@
...
-            arguments_.index_angle = neutral_angle_;
-            arguments_.thumb_angle = max_angle_;
-            return arguments_;
+            return arguments;
         }
 
+        static hand_arguments ClawMode(hand_arguments arguments)
+        {
+
+        }
```
Comment (Coreyboy1820) [inline subsystem-arm/implementations/hand/hand-modes.hpp:40]: defeats the purpose of 5 finger hand.

**Review by Coreyboy1820 — APPROVED** [review]

File: subsystem-arm/implementations/hand/hand-modes.hpp (hunk 4 above)
Comment (Mchan2003) [reply subsystem-arm/implementations/hand/hand-modes.hpp:40]: Can you explain more about why this mode defeats the purpose of 5 finger hands?

**Pull request merged**